src/main/scala/Server.scala

```diff
@@ -102,46 +70,44 @@ object Server extends App {
         }
 
       case Failure(error: SyntaxError) ⇒
-        complete(BadRequest, JsObject(
+        complete(ToResponseMarshallable(BadRequest → JsObject(
```

Comment:
I also had this issue before. Type inference is unable to figure out the types, so I also did it explicitly, just like you did here.